src/rx.cpp

```diff
@@ -432,6 +445,8 @@ int Aggregator::rx_ring_push(void)
     fprintf(stderr, "Override block 0x%" PRIx64 " flush %d fragments\n", rx_ring[rx_ring_front].block_idx, rx_ring[rx_ring_front].has_fragments);
 #endif
 
+    WFB_LOG(LogLevel::DEBUG, "Override block 0x%" PRIx64 " flush %d fragments", 
```

Comment:
@vertexodessa Need to remove `#if 0` above because it duplicated by this log message

Reply:
@svpcom fixed